sql/5_0_1-to-5_0_2_upgrade.sql

```diff
@@ -705,6 +705,10 @@ ALTER TABLE `facility` ADD `mail_zip` VARCHAR(10) default NULL;
 ALTER TABLE `facility` ADD `oid` VARCHAR(255) NOT NULL DEFAULT '' COMMENT 'HIEs CCDA and FHIR an OID is required/wanted';
 #EndIf
 
+#IfMissingColumn facility weno_id
+ALTER TABLE `facility` ADD `weno_id` int(e) default NULL;
+#EndIf
+
 #IfNotTable keys
```

Comment:
hi @juggernautsei , only need to add this in the 502 -> 503 script below

Reply:
Removed